README.md

```diff
@@ -1,5 +1,7 @@
 # Getting Started with Create React App
 
```

Comment:
- [ ] Please kindly update your readme file to reflect the changes for this milestone, checkout this [template ](https://github.com/microverseinc/readme-template)from Microverse and adapt it to your project. This will make your readme file more informative and professional.